tests/system/test_system.py

```diff
@@ -534,6 +534,7 @@ def test_handlers_w_extras(self):
         extra = {
             "trace": "123",
             "span_id": "456",
+            "trace_sampled": True,
```

Comment:
nit: do you want to add a regression test (sampled=false or omitted)?

Reply:
good point, I added a couple more assertions to this file